.travis.yml

```diff
@@ -28,20 +31,45 @@ matrix:
         - MAKE_TEST_TARGET=lint
     - env:
         - MAKE_TEST_TARGET=go_test
+
+    # pr branch deployments
+    - if: type = pull_request
+      env:
+      - DEPLOY_PR_STAGING_TARGET=webapp
+    - if: type = pull_request
+      env:
+      - DEPLOY_PR_STAGING_TARGET=results-processor
+    - if: type = pull_request
+      env:
+      - DEPLOY_PR_STAGING_TARGET=revisions/service
+    - if: type = pull_request
+      env:
+      - DEPLOY_PR_STAGING_TARGET=api/spanner/service
```

Comment:
We don't need this anymore, right? Let's stop deploying it.

Reply:
Done.